vms/proposervm/post_fork_option.go

```diff
@@ -121,6 +121,7 @@ func (b *postForkOption) buildChild(ctx context.Context) (Block, error) {
 		ctx,
 		parentID,
 		b.Timestamp(),
+		b.Height(),
```

Comment:
Why did we change this signature to take in the height? (we already have the height internally)